dsmr_parser/obis_references.py

```diff
@@ -43,6 +43,9 @@
 INSTANTANEOUS_CURRENT_L1 = r'^\d-\d:31\.7\.0.+?\r\n'
 INSTANTANEOUS_CURRENT_L2 = r'^\d-\d:51\.7\.0.+?\r\n'
 INSTANTANEOUS_CURRENT_L3 = r'^\d-\d:71\.7\.0.+?\r\n'
+FUSE_THRESHOLD_L1 = r'^\d-\d:31\.4\.0.+?\r\n'  # Applicable when current limitation is active
+FUSE_THRESHOLD_L2 = r'^\d-\d:51\.4\.0.+?\r\n'  # Applicable when current limitation is active
+FUSE_THRESHOLD_L3 = r'^\d-\d:71\.4\.0.+?\r\n'  # Applicable when current limitation is active
```

Comment:
Maybe these should be added in a separate PR?

Reply:
It's because this PR just contains 2 commits. One is the mbus change and the other is #139 . Just for the ease to not need to rebase after merge of the first PR.

Reply:
I do not agree this is right. It is just not clean as you do not need that commit here. It that were the case then this PR should be a draft.